build-package.sh

```diff
@@ -1,25 +1,33 @@
 #!/bin/bash
 
 DIR=$(mktemp -d)
+HUGO_VERSION=0.17
+
 cd $DIR
 
 # Install awscli python lib + deps in buildroot
-pip install awscli -t .
+pip install pyyaml awscli -t .
+
 # Fetch aws cli wrapper from github
-wget https://raw.githubusercontent.com/aws/aws-cli/1.9.15/bin/aws \
+wget https://raw.githubusercontent.com/aws/aws-cli/develop/bin/aws \
```

Comment:
To be honest, I don't know the strategy. master is 61 commits ahead of 'develop' 😕